references/cicd-sharedflow-pipeline/test/integration/pom.xml

```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
as above - we shouldnt need two poms, should we? for simplification we only need one.

Reply:
Wanted to keep them separate so that it can be called separately. As you can see in pipeline.sh and README, we are calling that with `-f` so that its outside and can be altered if needed

Reply:
you dont need a separate pom for that, you can just call the goal on its own. ref: https://stackoverflow.com/questions/3166538/how-to-execute-maven-plugin-execution-directly-from-command-line

Reply:
@laughingbiscuit / @danistrebel - merging them to one pom will make it complicated. For eg, the pom supplies the name of the proxy/sharedflow, copy the source to target, replace, etc. So I will have to externalize all of those as arguments and supply them during the Maven call. Does it make sense to do all that or just leave it as 2 poms. I agree its different from other repos. But it just makes it more cleaner. Please advice.

Reply:
it's a bit of a trade-off. I see the point of only having a single pom to reduce boilerplate. But the pom are relatively different and merging them will just externalize the difference in configuration. I'd probably stick to two poms but clean out everything that isn't needed for this reference. E.g. config plugin isn't needed on the shared flow as I can see. I'd also remove some of the config parameters that aren't needed in this case.

Reply:
Done

Reply:
sounds like a good compromise, thanks both